apps/e2e-tests/tests/frontend/terms-and-conditions.test.ts

```diff
@@ -0,0 +1,9 @@
+import { test, expect } from '@playwright/test';
+
+test('basic test', async ({ page }) => {
+  await page.goto('https://1155.hub.asap.science/terms-and-conditions');
+
```

Comment:
I think this test will fail once this is merged. 
`https://1155.hub.asap.science/` won't be available

Reply:
thanks @ana-yldio I guess i should have marked this as WIP :D